docs/proposals/compute-resource-metrics-api.md

```diff
@@ -115,13 +116,11 @@ only be available through the kubelet. The types of metrics are detailed
 Additionally, all endpoints (except root discovery endpoint) support the
 following optional query parameters:
 
-- `start` - start time to return metrics from; type json encoded
-  `time.Time`; since samples are retrieved at discrete intervals, the first
-  sample after the start time is the actual beginning.
-- `end` - end time to return metrics to; type json encoded `time.Time`
-- `step` - the time step between each stats sample; type int (seconds), default
```

Comment:
If we only have a poll api (for now) then a step bigger than a sampling window should mean that only 1 sample is returned.